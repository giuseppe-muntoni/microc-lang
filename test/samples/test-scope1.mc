float x; 

void main() {
    x = 2.5;
    print_float(x);
    int x;
    x = 0;
    print(x);
}