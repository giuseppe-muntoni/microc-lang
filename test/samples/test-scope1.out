2.5000
0
